src/components/Books.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Book from './Book';
+
+export default function Books() {
+  const books = [
+    {
+      title: 'The Hunger Games',
+      author: 'Suzanne Collins',
+      category: 'Action',
+      length: 27,
+      chapter: '17',
+    },
+    {
+      title: 'Dune',
+      author: 'Frank Herbert',
+      category: 'Science Fiction',
+      length: 38,
+      chapter: 'Chapter 3: "A Lesson Learned"',
+    },
+    {
+      title: 'Capital in the Twenty-First Century',
+      author: 'Suzanne Collins',
+      category: 'Economy',
+      length: 27,
+      chapter: 'Introduction',
+    },
+  ];
+  return (
+    <section>
+
+      <div>
+        {books.map((book) => (
+          <Book key={book.id} title={book.title} author={book.author} category={book.category} />
+        ))}
+      </div>
+      <form>
+        <input type="text" />
+        <input type="text" />
+        <button type="submit" label="submit" />
+      </form>
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md)(`Create a separate component for this form (with inputs for a title and an author`) for this project, and best practice, creating reusable components, please it will be nice to create a separate component for the form, then import and render it in this `Books.js` components.